packages/main/src/ListItemBase.ts

```diff
@@ -112,6 +112,9 @@ class ListItemBase extends UI5Element implements ITabbable {
 		}
 	}
 
+	@property({ type: Boolean, defaultValue: false })
```

Comment:
Did you try to style the last list item in a Card with
 ::slotted(*:last-child) {
}
or it does not work


*btw, all boolean props are false by default, no nee to write "defaultValue: false"